taxi/models.py

```diff
@@ -15,7 +16,20 @@ def __str__(self):
 
 
 class Driver(AbstractUser):
-    license_number = models.CharField(max_length=255, unique=True)
+    license_number = models.CharField(
+        max_length=8,
+        unique=True,
+        validators=[
+            RegexValidator(
+                regex=r"^[A-Z]{3}\d{5}$",
+                message=(
+                    "Input format is incorrect, "
+                    "please follow the example: "
+                    "`AAA12345`"
+                ),
+            )
```

Comment:
Move this to form checking, since you won't be able to customize drivers and license numbers via admin panel.
Also better not to use regex, it's ok, but it's not explicit.